.template/addons/github/.github/README.md.tt

```diff
@@ -0,0 +1,16 @@
+# Github Actions
+
+## Requirements:
+
+- Generate new token on docker registry, support `docker.io`, `ghcr.io`, `azurecr.io`, `gcr.io`, `registry.gitlab.com`
+- Generate new Heroku API key
+
+## Setup the CI
+
+### Secrets
+
+Setup the following secrets in order to run the workflows:
+
+- DOCKER_REGISTRY_HOST
+- DOCKER_REGISTRY_TOKEN
```

Comment:
How about `DOCKER_REGISTRY_USERNAME` and `DOCKER_IMAGE` @hoangmirs ?

Reply:
Currently, we're using `github.repository_owner` variable for `DOCKER_REGISTRY_USERNAME` and `github.repository` for `DOCKER_IMAGE`.
And I've set them in the workflows already, so I didn't put them here

Reply:
I see @hoangmirs but it hard to get it, also in case the project is using Docker hub then the `github.repository_owner` for user name might not work, maybe we could mention it here like

```
Setup the following secrets in order to run the workflows:

- DOCKER_REGISTRY_HOST
- DOCKER_REGISTRY_TOKEN
- DOCKER_REGISTRY_USERNAME (pre-setup is ${{ github.repository_owner }})
- DOCKER_IMAGE (pre-setup is ${{ github. repository }})
- HEROKU_API_KEY (for deployment)
```

Reply:
also I'm not sure this README file will be added into to the project :D